styles/index.css

```diff
@@ -0,0 +1,114 @@
+:root {
+    --main-bg-color: rgb(16, 3, 46);
+    --starwars-yellow: rgb(88%, 69%, 0%);
+    --logo-width: 150px ;
+}
+
+/* BACKGROUND */
+body {
+    background-color: var(--main-bg-color);
+    margin: 0;
+}
+
+
+.main-bg-image {
+    width: 100%;
+    height: 170%;
+    position: absolute;
+    background-position: center;
+    background-image: url("/images/background_stars.webp");
+    background-size: cover;
+    background-repeat: no-repeat;
+    backdrop-filter: var(--main-bg-color);
+    left: 0;
+}
+
+#main-grid-container {
+    max-width: 1440px;
+    display: grid;
+    align-items: center;
+    justify-items: center;
+    grid-template-columns: 1fr 5fr;
+    grid-template-rows: auto 1fr;
+    grid-template-areas:
+        "logo     navbar "
+        "content content ";
+}
+
+.sw-logo {
```

Comment:
Logo at this size and placement looks out of place atm. Maybe make it bigger and more close to the navbar.

Reply:
Please consult this with each other and decide if it'll be implemented as part of this task or added to future suggestions. If you decide to change it as part of this task, please do this as soon as you can. If not, please let the PM know that.

Reply:
I little changed position and I made logo bigger.